src/components/Modals/SelectTestnetRPC/SelectTestnetRPC.tsx

```diff
@@ -30,7 +30,7 @@ export const SelectTestnetRPC: React.FC<ISelectTestnetRPC> = ({
   )
 
   const isAddressValid = () => {
-    const urlRegex = /^https?:\/\/[^.]+\.[^.]+/
+    const urlRegex = /^wss?:\/\/[^.]+\.[^.]+/
```

Comment:
IMO should be `wss` | `https` | `http`

Reply:
`initPolkadotApi` accepts only `ws` and `wss`, so how about `ws | wss`?

Reply:
ok